URLDemonstrationAPP/app.R

```diff
@@ -14,8 +14,9 @@ ui <- navbarPage(
              sidebarLayout(
                sidebarPanel( 
                  textInput("url", 
-                           p("Url:"), 
-                           value = "")),
+                           p("Url:"),
+                           value = "http://labs.statsbiblioteket.dk/labsapi/api/aviser/export/fields?query=revolution%20AND%20py%3A%5B1840%20TO%201880%5D&fields=link&fields=recordID&fields=timestamp&fields=pwa&fields=cer&fields=fulltext_org&fields=pageUUID&fields=editionUUID&fields=titleUUID&fields=editionId&fields=familyId&fields=newspaper_page&fields=newspaper_edition&fields=lplace&fields=location_name&fields=location_coordinates&max=1&structure=header&structure=content&format=CSV",
+                           placeholder = "copy URL here")),
```

Comment:
Even though this is a solution I think it might cause more confusion on where, what these little statistics are. I think a better idea would be adding an action button and then leaving the value of the textInput empty with the placeholder saying some like "Paste URL here". To read more about buttons in Shiny see [Actions Buttons](https://shiny.posit.co/r/articles/build/action-buttons/). _Pattern 2 - Delay reactions_ shows an example thats pretty close to what I imagined